jina/serve/runtimes/asyncio.py

```diff
@@ -148,7 +148,9 @@ def is_ready(ctrl_address: str, **kwargs) -> bool:
         """
 
         try:
-            GrpcConnectionPool.send_request_sync(ControlRequest('STATUS'), ctrl_address)
+            GrpcConnectionPool.send_request_sync(
+                ControlRequest('STATUS'), ctrl_address, timeout=1.0
```

Comment:
why is here `timeout` hardcoded to 1?

Reply:
We use this part during the ready check of the Pods, which is controlled by the `timeout_ready`. The way it is implemented atm is that we try the STATUS request multiple times until `timeout_ready` is hit. What we use as the timeout for the STATUS request itself here does not really matter. I think 1 second is okay, it could also be 0.5 or 2 seconds.

Reply:
Understand